packages/slice-machine/src/features/onboarding/content.tsx

```diff
@@ -0,0 +1,115 @@
+import { Text } from "@prismicio/editor-ui";
+
+import type {
+  OnboardingStep,
+  OnboardingStepContentDefinition,
+  OnboardingStepType,
+} from "@/features/onboarding/types";
+
+export const onboardingSteps: OnboardingStep[] = [
+  {
+    id: "createPage",
+    title: "Create Your Page",
+    description: "Setup a new page.",
+  },
+  {
+    id: "codePage",
+    title: "Code your page",
+    description: "Build your page's structure",
+  },
+  {
+    id: "addSlice",
+    title: "Add a Slice",
+    description: "Insert reusable sections.",
+  },
+  {
+    id: "pushModels",
+    title: "Push Your Models",
+    description: "Deploy your models.",
+  },
+  {
+    id: "writeContent",
+    title: "Write Content",
+    description: "Add engaging content.",
+  },
+];
+
+type GetOnboardingStepsContentProps = {
+  repositoryUrl: string;
+};
+
+export const getOnboardingStepsContent = ({
+  repositoryUrl,
+}: GetOnboardingStepsContentProps): Record<
+  OnboardingStepType,
+  OnboardingStepContentDefinition
+> => ({
+  createPage: {
+    videoUrl:
+      "https://res.cloudinary.com/dmtf1daqp/video/upload/v1700213517/IN-APP-GUIDE-SM/ADD_SLICE.mp4",
+    content: () => (
+      <>
```

Comment:
Shouldn't these be <Text>?

Reply:
Be Text?

Reply:
yeah, i promise you i wrote `<Text>` without the code formatting and when i posted github editor must have drop it - I'm not talking about the content of the component, but the component itself, not sure what you mean with placeholder here, i understand that the text won't be final, but i guess we want to use our own font and styling, right?

Reply:
Not sure I'm following 😕 Can you give a code example?

Reply:
it's ok, i saw that you already fixed it https://github.com/prismicio/slice-machine/pull/1399/commits/76f217b8fd65a4fb8605bacbecda9cc58814dcd3

Reply:
Oh ok, so I understood it correctly 😄